sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 # Release History
 
-## 1.0.0-beta.10 (Unreleased)
+## 1.0.0 (2021-06-04)
 
+### Bug Fixes
+
+- Make `RequestFailedException` copiable so it can be propagated across thread.
```

Comment:
```suggestion
- Make `RequestFailedException` copyable so it can be propagated across thread.
```

Reply:
Both are fine, in fact, I think I have seen `copiable` more often.

Reply:
https://en.wiktionary.org/wiki/copiable

https://english.stackexchange.com/questions/545335/why-do-some-spell-checker-mark-copiable-as-an-error